lib/game.js

```diff
@@ -187,8 +171,9 @@ function isObstacle(collider){
   return collider instanceof Obstacle;
 }
 
-function isPowerUp(collider){
-  return collider instanceof PowerUp;
+function crash(){
+  $('.game-canvas').hide();
+  $('.end-game').show();
 }
 
```

Comment:
I think that it was a good Idea to pull this out into a private method since it does not need to be called on a specific object.